includes/admin/tools.php

```diff
@@ -1377,7 +1377,18 @@ function edd_tools_sysinfo_get() {
 			continue;
 
 		$update = ( array_key_exists( $plugin_path, $updates ) ) ? ' (needs update - ' . $updates[$plugin_path]->update->new_version . ')' : '';
-		$return .= $plugin['Name'] . ': ' . $plugin['Version'] . $update . "\n";
+		$plugin_url = '';
+		if ( ! empty( $plugin['PluginURI'] ) ) {
+			$plugin_url = $plugin['PluginURI'];
+		} elseif ( ! empty( $plugin['AuthorURI'] ) ) {
+			$plugin_url = $plugin['AuthorURI'];
+		} elseif {
```

Comment:
```suggestion
		} elseif ( ! empty( $plugin['Author'] ) ) {
```
The `elseif` has to have something for the `if` please--on all three of these.